journal-of-mammalogy.csl

```diff
@@ -140,9 +140,11 @@
       </if>
       <else-if type="bill book chapter graphic legal_case legislation motion_picture paper-conference  song" match="any">
         <group prefix=". " delimiter=" ">
-          <label variable="page" form="short" text-case="capitalize-first" suffix=" "/>
-          <text variable="page"/>
-          <text term="in"/>
+          <group delimiter=" ">
+            <label variable="locator" form="short" text-case="capitalize-first"/>
+            <text variable="locator"/>
+            <text term="in"/>
+          </group>
```

Comment:
I made the "in" term conditional by placing it in a group, and switched from "page" to "locator". Didn't really dig into the style guide, but seemed like a reasonable correction.